bindings/python/src/lib.rs

```diff
@@ -84,6 +103,15 @@ fn normalizers(_py: Python, m: &PyModule) -> PyResult<()> {
 /// Tokenizers Module
 #[pymodule]
 fn tokenizers(_py: Python, m: &PyModule) -> PyResult<()> {
+    // Register the fork callback
+    #[cfg(target_os = "linux")]
+    unsafe {
+        if !REGISTERED_FORK_CALLBACK {
+            libc::pthread_atfork(None, None, Some(child_after_fork));
```

Comment:
💯 💯 🎉 🎉 !!!

Reply:
TIL

Reply:
I didn't know about it either, and it seems to be working really well. This should let us have a very flawless experience. :tada: